### ISSUE TITLE
enet-coco.cfg bug?

### ISSUE THREAD

**Issue by dccho** [AlexeyAB/darknet#4067]
In last yolo layer, the value of the mask parameter is '1,2,3' not '0,1,2'. It is supposed to start from 0 not 1. In previous yolo layer, the value of the mask parameter is '3,4,5'. It means that two yolo layers share mask 3.

**Comment by AlexeyAB** [comment]
Yes, there is a bug: https://github.com/AlexeyAB/darknet/blob/master/cfg/enet-coco.cfg#L1064

**Comment by WongKinYiu** [comment]
Both of enet-coco.cfg and yolov3-tiny-prn.cfg are trained using masks '1,2,3' and '3,4,5'.
The reason is for fair comparison with original yolo-v3-tiny.cfg https://github.com/pjreddie/darknet/commit/481d9d98abc8ef1225feac45d04a9514935832bf#r28887402
The result and weight file of yolo-v3-tiny provided by the author is trained using this setting.

You can just replace it to '0,1,2' and '3,4,5' if you want.

**Comment by VideoPlayerCode** [comment]
@dccho @AlexeyAB Yes there is a bug and we talked in detail about it here: https://github.com/WongKinYiu/PartialResidualNetworks/issues/2
